.github/workflows/swift-toolchain.yml

```diff
@@ -2088,7 +2088,13 @@ jobs:
         run: cmake --build ${{ github.workspace }}/BinaryCache/sourcekit-lsp --target install
       - name: Install swift-format
         run: cmake --build ${{ github.workspace }}/BinaryCache/swift-format --target install
-
+      - name: Install Swift Toolchain
+        uses: compnerd/gha-setup-swift@8f43ccc3e8bac89829862af09de9567c807c1c12
+        with:
+          github-repo: thebrowsercompany/swift-build
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          release-asset-name: installer-amd64.exe
+          release-tag-name: '20231016.0'
```

Comment:
Please don't do this - we want to use the just built Swift for building swift-inspect.

Reply:
What is the path to the just-built swift, and what jobs must be complete before I can run this job using that swift?

Reply:
That depends on the pipeline configuration. See some of the other builds in this stage for the path.

This stage should have everything built - you just need the compilers and SDK built. For swift-inspect, you will also need SPM, but that was just built in this stage.